src/js/components/Form/__tests__/__snapshots__/Form-test-uncontrolled.js.snap

```diff
@@ -1941,6 +1941,7 @@ exports[`Form uncontrolled reset clears select, checkbox, radiobuttongroup 1`] =
 }
 
 <div
+  aria-hidden="false"
```

Comment:
Why are we adding this? It seems like it isn't needed.

Reply:
Good call, let me look into it.

Reply:
Resolved.